packages/amplify-graphql-schema-generator/API.md

```diff
@@ -258,20 +277,6 @@ export class MySQLDataSourceAdapter extends DataSourceAdapter {
     test(): Promise<boolean>;
 }
 
-// @public (undocumented)
-export interface MySQLDataSourceConfig {
-    // (undocumented)
-    database: string;
-    // (undocumented)
-    host: string;
-    // (undocumented)
-    password: string;
-    // (undocumented)
-    port: number;
-    // (undocumented)
-    username: string;
-}
-
```

Comment:
Currently outside of API repo, Gen2 generate command is the only package that relies on this package. Generate command doesn't use this interface so it is safe to remove it. Same for the `PostgresDataSourceConfig`.